### PR TITLE
Rviz overlay render fix

### PR DESCRIPTION
**Pull request by adamdbrw**
Restores MFD superclass for SteeringAngle plugin. 

Moves rendering to update() function called from the Display superclass in the rendering thread. This fixes the issue.

Note that other plugins can be similarly modified.

### CODE REVIEW

File: common/util/rviz_plugins/autoware_vehicle_rviz_plugin/src/tools/steering_angle.cpp [hunk 1]
```diff
@@ -94,29 +94,15 @@ SteeringAngleDisplay::~SteeringAngleDisplay()
 
 void SteeringAngleDisplay::onInitialize()
 {
-  RTDClass::onInitialize();
+  MFDClass::onInitialize();
```
Comment (mitsudome-r) [inline common/util/rviz_plugins/autoware_vehicle_rviz_plugin/src/tools/steering_angle.cpp:97]: I think I said that we can switch back to MFDClass in the previous meeting, but after having second thought, I think we can keep it as RTDClass. The change on moving visualization process to `update` function from `processMessage` can still be applied for stability and process time.
I apologize for going back and forth, but could you make the super class to RTD?

**Review by mitsudome-r — COMMENTED** [review]
Thanks for the review. I have made a comment about the super class.

File: common/util/rviz_plugins/autoware_vehicle_rviz_plugin/src/tools/steering_angle.cpp (hunk 1 above)
Comment (adamdbrw) [reply common/util/rviz_plugins/autoware_vehicle_rviz_plugin/src/tools/steering_angle.cpp:97]: in all the plugins?

**Review by mitsudome-r — APPROVED** [review]
LGTM

**Pull request merged**